src/preload.py

```diff
@@ -31,9 +31,11 @@
 
 IMAGE = "/img/resin.img"
 
+# In bytes:
 SECTOR_SIZE = 512
 MBR_SIZE = 512
 GPT_SIZE = SECTOR_SIZE * 34
+GRUB_SIZE = 446
```

Comment:
To be pedantic, perhaps this should be `MBR_BOOTSTRAP_CODE_SIZE` ?

Looking at https://en.wikipedia.org/wiki/Master_boot_record#Sector_layout I guess if we copy the first 446 bytes, we'll also be copying the disk-signature. ISTR that Linux uses the disk-signature to generate/store the disk's UUID values, so I guess this means we'd be creating new images with identical UUIDs?
I've got no idea if that's something we want to be doing or not, just pointing ti out ;-)

Reply:
Good catch for the name, i amende my commit.
About the uuid generation, resin-preload is not supposed to update the disk uuid so that's fine.